app/api/dao/mentorship_relation.py

```diff
@@ -365,3 +365,29 @@ def list_pending_mentorship_relations(user_id):
                 pending_requests += [relation]
 
         return pending_requests, 200
+
+    @staticmethod
+    @email_verification_required
+    def count_pending_requests(user_id):
+        """Returns a number of pending requests.
+
+        Args:
+            user_id: ID of the user counting all pending relationships.
+
+        Returns:
+            message: A message corresponding to the completed action;
+            success if pending mentorship relation requests' count is returned,
+            failure if otherwise.
+        """
+
+        user = UserModel.find_by_id(user_id)
+        now_timestamp = datetime.now().timestamp()
+        pending_requests = 0
+        all_relations = user.mentor_relations + user.mentee_relations
+
+        for relation in all_relations:
```

Comment:
have you ever tried lamda expressions with python?

Reply:
not really. I will Google it now. Or maybe you could explain that to me?

Reply:
@annabauza could you say me, what exactly do you want to do with a lambda in this function?

Reply:
I know what a lambda is, but I don't know exactly how to use it here.

Reply:
I guess, I understand what you want to do. Could you give me a minute?

Reply:
@annabauza, I used lambda here.